core/templates/pages/preferences-page/form-fields/preferred-languages.component.spec.ts

```diff
@@ -89,8 +89,7 @@ describe('Preferred Languages Component', () => {
 
   it('should filter choices when search query is non-empty', () => {
     const mockChoices = [
-      { id: 'en', text: 'English' },
-      { id: 'fr', text: 'French' }
```

Comment:
@Adarsh7825 why would you remove this option? This reduces the usefulness of the test. Now we cannot verify if the logic to filter choices is working correctly or not

Reply:
Done